README.adoc

```diff
@@ -249,26 +254,31 @@ You can find more detailed explanations on how to define your own system propert
 them via the `System.getProperty` call in the
 https://openliberty.io/guides/microprofile-intro.html[MicroProfile guide].
 
-=== Testing the processing with JSON-P
-
-Write the `testArtistCount` test case. This test checks that deserialization with JSON-P was done correctly
-and that the correct number of artist is returned.
+=== Testing the binding process
+For your test classes to have access to JSON-B, add the following dependency to your
+`pom.xml` file, which is already done for you:
 
-[source, java, indent = 0]
+[source, xml]
 ----
-include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java[tags=testArtistCount]
+<dependency>
+    <groupId>org.eclipse</groupId>
+    <artifactId>yasson</artifactId>
+    <version>1.0.1</version>
+    <scope>test</scope>
+</dependency>
 ----
 
-Write the `assertResponse` helper method that asserts that the response code you receive is valid (200):
+Write the `testArtistDeserialization` test case.  This test checks that Artist instances created from
+the REST data and those that are hardcoded perform the same.
```

Comment:
- Remove extra space on line 271.
- Earlier, I saw "Artist" formatted as `Artist`. Should we use the code phrase format on line 271 also?